src/lm/arpa-file-parser.cc

```diff
@@ -147,6 +155,7 @@ void ArpaFileParser::Read(std::istream &is, bool binary) {
     int32 ngram_count = 0;
     while (++line_number_, getline(is, current_line_) && !is.eof()) {
       if (current_line_.empty()) continue;
+      current_line_ = rtrim(current_line_);
```

Comment:
Adding this line will incur a big performance penalty and I don't think it's necessary to solve the current issue.  The ARPA file format is not well defined, but I don't think a reasonable person would put leading space before the \1-grams:  marker.